lua/entities/gmod_wire_relay.lua

```diff
@@ -44,23 +44,23 @@ function ENT:Setup(keygroup1, keygroup2, keygroup3, keygroup4, keygroup5, keygro
 	self.throws 		= throws
 	self.nokey			= nokey
 	
-	//build inputs and putputs, init all nil values
+	-- build inputs and putputs, init all nil values
 	for p=1, self.poles do
 		self.outputs[p] = outpoles[p]
 		self.Value[p] = self.Value[p] or 0
 		for t=1, self.throws do
-			//inputs[ p * self.poles + t ] = t .. outpoles[p]
+			-- inputs[ p * self.poles + t ] = t .. outpoles[p]
```

Comment:
i use "--" without a space for commented out code and "--" with a space for the rest